xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -237,39 +237,39 @@
 ##              DISPLAY CONFIRM MESSAGE
 #######################################################
 #macro(displayConfirmMessage)
-  #if($xwiki.hasRecycleBin())
-    #if($services.refactoring.isAllowedToSkipTheRecycleBin())
-      <div class="xform">
-        <dl>
+  <div class="xform">
+    <dl>
+      #if($xwiki.hasRecycleBin())
+        #if($services.refactoring.isRecycleBinSkippingAllowed())
           <dt>
-            $escapetool.xml($services.localization.render('core.recyclebin.wantToSkip'))
+            $escapetool.xml($services.localization.render('core.recycleBin.shouldSkip'))
           </dt>
           <dd>
             <label class="xwiki-form-listclass">
-              <input name="skipRecycleBin" type="radio" value="false" checked/>
-              $escapetool.xml($services.localization.render('core.recyclebin.wantToSkip.no'))
+              <input name="shouldSkipRecycleBin" type="radio" value="false" checked/>
+              $escapetool.xml($services.localization.render('core.recycleBin.shouldSkip.no'))
             </label>
             <label class="xwiki-form-listclass">
-              <input name="skipRecycleBin" type="radio" value="true"/>
-              $escapetool.xml($services.localization.render('core.recyclebin.wantToSkip.yes'))
+              <input name="shouldSkipRecycleBin" type="radio" value="true"/>
+              $escapetool.xml($services.localization.render('core.recycleBin.shouldSkip.yes'))
             </label>
           </dd>
-        </dl>
-      </div>
-    #else
-      #if($hasInlinks)
-        <p><strong>$escapetool.xml($services.localization.render('core.recyclebin.confirmWithInlinks'))</strong></p>
+        #else
+          #if($hasInlinks)
+            <dt>$escapetool.xml($services.localization.render('core.recyclebin.confirmWithInlinks'))</dt>
+          #else
+            <dd>$escapetool.xml($services.localization.render('core.recyclebin.confirm'))</dd>
```

Comment:
I still don't know what this means. Did I miss a reply?

Reply:
`core.recyclebin.confirmWithInlinks` is used when the page has backlings, otherwise `core.recyclebin.confirm` is used.

Reply:
yes but my question is what is the difference in the message for confirming a deletion has to do with that?

Reply:
ok I know you just kept what was there... :)  Just that since you changed it, maybe there was also something to improve/fix